test/cutting/test_find_cuts.py

```diff
@@ -46,6 +47,24 @@ def test_find_cuts(self):
             assert len(metadata["cuts"]) == 2
             assert {"Wire Cut", "Gate Cut"} == cut_types
             assert np.isclose(127.06026169, metadata["sampling_overhead"], atol=1e-8)
+            assert metadata["min_reached"]
```

Comment:
This is a stronger condition, as the way you previously wrote it would pass if `metadata["min_reached"]` is any ["truthy" value](https://docs.python.org/3/library/stdtypes.html#truth-value-testing).

```suggestion
            assert metadata["min_reached"] is True
```

Reply:
I'll double check but I think the reason why I didn't do this earlier was because I was getting a linting error.

Reply:
Nevermind, I think I was hallucinating there. I have made the changes discussed here ([53deb86](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/586/commits/53deb866f797d69868e59ec87091747af5f2a309)).

Reply:
> I was getting a linting error.

I wouldn't be surprised if you saw this elsewhere when using something like this.  In general, it's much better to e.g. condition a code block on truthiness rather than `is True`; but this is one of those special cases (in a test) where it actually makes sense to make sure that it `is True`.